packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.routing.js

```diff
@@ -52,11 +52,7 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
 
-      isZoneValid: /* @ngInject */ (newDnsZone) => (name) =>
-        newDnsZone
-          .getZoneNameValidation(name)
-          .then(() => true)
-          .catch(() => false),
+      isZoneValid: /* @ngInject */ () => Promise.resolve(true),
```

Comment:
Always ?

Reply:
always

Reply:
As `isZoneValid` is only used in component's binding (https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/dns-zone/new/new.component.js#L8)
maybe we could remove `checkZoneName` & `zoneNameisValid` statements in controller and template (https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.controller.js#L49 & https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.html#L44)

Reply:
Good point @cbourgois!

Will take this patch as is in order to create a new release but we will apply your suggestion via a separate Pull Request.
Thanks!